requirements/edx/base.txt

```diff
@@ -76,18 +76,18 @@ django-ratelimit==3.0.1   # via -r requirements/edx/base.in
 django-require==1.0.11    # via -r requirements/edx/base.in
 django-sekizai==2.0.0     # via -r requirements/edx/base.in, django-wiki
 django-ses==2.0.0         # via -r requirements/edx/base.in
-django-simple-history==2.12.0  # via -r requirements/edx/base.in, edx-enterprise, edx-organizations, ora2
+django-simple-history==3.0.0  # via -c https://raw.githubusercontent.com/edx/edx-lint/master/edx_lint/files/common_constraints.txt, -r requirements/edx/base.in, edx-enterprise, edx-organizations, ora2
```

Comment:
@igobranco @MaferMazu Running `make compile-requirements` updated this package. Should we proceed with this change or should I manually change it?

Reply:
I think we should use a safer approach. 
Please review the PR to still use the `2.12.0` version.

Reply:
Ready